src/main/java/listeners/OnlinerTestListener.java

```diff
@@ -1,25 +1,27 @@
 package listeners;
 
+import com.automation.remarks.testng.UniversalVideoListener;
 import com.google.common.collect.ImmutableMap;
-import io.qameta.allure.Attachment;
 import org.apache.commons.io.FileUtils;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 import org.testng.ITestContext;
 import org.testng.ITestListener;
 import org.testng.ITestResult;
+import org.testng.annotations.Listeners;
+import util.AllureUtil;
 import java.io.File;
-import java.io.FileOutputStream;
 import java.io.IOException;
-
 import static com.github.automatedowl.tools.AllureEnvironmentWriter.allureEnvironmentWriter;
 
 /***
  * OnlinerTestListener is used to log information on the test run flow and to the reporting system.
  */
+@Listeners(UniversalVideoListener.class)
 public class OnlinerTestListener implements ITestListener {
 
     protected final Logger logger = LogManager.getLogger(this);
+    AllureUtil allureUtil = new AllureUtil();
```

Comment:
Use modifier

Reply:
utils classes are supposed to have static methods so that you don't have to create there objects.

Reply:
ok